From f28db0bbeaca953bfac1466445db6e09930e9931 Mon Sep 17 00:00:00 2001
From: yuyue <yuyue@mprc.pku.edu.cn>
Date: Thu, 12 Jan 2012 16:55:52 +0800
Subject: [PATCH 196/641] UniCore64: replace EXTRA_FLAGS with ccflags-y

EXTRA_CFLAGS is still supported but its usage is deprecated.

Signed-off-by: yuyue <yuyue@mprc.pku.edu.cn>
---
 arch/unicore64/boot/Makefile |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/unicore64/boot/Makefile b/arch/unicore64/boot/Makefile
index 3fc095b..c3b55cc 100644
--- a/arch/unicore64/boot/Makefile
+++ b/arch/unicore64/boot/Makefile
@@ -2,7 +2,7 @@
 targets		:= Image piggy.bin boot.lds boot.bin zImage unicore64-fpga.dtb
 
 # Do not recognize built-in functions in uncompress.c
-EXTRA_CFLAGS	:= -fno-builtin
+ccflags-y	:= -fno-builtin
 
 # Only copy text segment from boot.bin to zImage
 OBJCOPYFLAGS	:= -O binary -S
-- 
1.7.9.5

